filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
+    params = list(args)
+    params.append(a)
+    it = iter(params)
+    return any(it) and not any(it)
+
+class FilterModule(object):
+    def filters(self):
+        return {
+            'only_one_true': only_one_true
```

Comment:
I think this could basically be a plain `any`:

    return {'any': any}

Which is the same as "only one true" 

https://docs.python.org/2/library/functions.html#any